### PR TITLE
chore(storage-browser): add e2e test for offline Create folder action

### PR DESCRIPTION
**Pull request by hbuchel**
#### Description of changes

- Adds an e2e test for Create folder to verify that it shows a network error if folder creation is attempted while offline.
- There are two files that just have a small prettier change each.

Screenshot 2024-09-06 at 12 30 53 PM

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx [hunk 1]
```diff
@@ -78,7 +78,9 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
   const [state] = useControl('NAVIGATE');
   const { path } = state;
 
-  const [{ data, isLoading, hasError }, handleList] = useAction('LIST_LOCATION_ITEMS');
```
Comment (hbuchel) [inline packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx:81]: Prettier change, though I'm not sure why it wasn't already there.

File: packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/__tests__/LocationDetailView.spec.tsx [hunk 2]
```diff
@@ -145,7 +145,7 @@ describe('LocationDetailView', () => {
       hasError: true,
       message: errorMessage,
       result: [{ key: 'test1', type: 'FOLDER' }],
-      nextToken: 'some-token'
```
Comment (hbuchel) [inline packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/__tests__/LocationDetailView.spec.tsx:148]: Prettier change, though I'm not sure why it wasn't already there.

File: packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx (hunk 1 above)
Comment (reesscot) [reply packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx:81]: I've been getting this a lot, there seems to be some non-deterministic behavior with our prettier setup

Comment (dbanksdesign) [reply packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx:81]: Make sure your VSCode (not you personally, but all of us) is using the project's version of prettier and not a globally installed one

**Review by dbanksdesign — APPROVED** [review]

File: packages/e2e/features/ui/components/storage/storage-browser/create-folder.feature [hunk 3]
```diff
@@ -38,3 +38,19 @@ Feature: Create folder with Storage Browser
     When I type a new "Enter folder name:" with value "Blackberry/"
     When I lose focus on "Enter folder name:" input
     Then I see "Folder name must be at least one character and cannot contain a \"/\"."
+
```
Comment (ashwinkumar6) [inline packages/e2e/features/ui/components/storage/storage-browser/create-folder.feature:41]: UI has some minor changes
- "Actions" -> "Menu Toggle"
- "Enter folder name:" -> "Folder name"

**Review by ashwinkumar6 — COMMENTED** [review]
Thx lgtm, UI has had some minor changes so we might want to re-run on latest

**Comment by ashwinkumar6** [comment]
changes added to #6206

**Pull request closed without merging**